Makefile

```
@@ -86,6 +86,14 @@ mod-outdated: ## check outdated dependencies
 authors: ## generate AUTHORS file from git history
 	scripts/docs/generate-authors.sh
 
+.PHONY: completion
+completion: binary
+completion: /etc/bash_completion.d/docker
+completion: ## generate and install the completion scripts
+
+/etc/bash_completion.d/docker: FORCE ## generate and install the bash-completion script
+	docker completion bash > /etc/bash_completion.d/docker
```

We should also make sure that the directory exists:
```suggestion
    mkdir -p /etc/bash_completion.d
	docker completion bash > /etc/bash_completion.d/docker
```

---

Good one; I wrote it with the dev-container in mind (which had it set up already), but doesn't hurt to add

---

updated 👍